pkg/api/validation/dynakube/eec.go

```diff
@@ -6,7 +6,8 @@ import (
 )
 
 const (
-	errorExtensionExecutionControllerImageNotSpecified = `DynaKube's specification enables the Prometheus feature, make sure you correctly specify the ExtensionExecutionController image.`
+	errorExtensionExecutionControllerImageNotSpecified       = `DynaKube's specification enables the Prometheus feature, make sure you correctly specify the ExtensionExecutionController image.`
+	errorExtensionExecutionControllerInvalidPVCConfiguration = "You provided a PVC and demanded to use an ephemeral volume at the same time for extension controller. These settings are mutually exclusive, please choose only one."
```

Comment:
The message sounds kinda weird:
```suggestion
	errorExtensionExecutionControllerInvalidPVCConfiguration = `DynaKube specifies a PVC for the extension controller while ephemeral volume is also enabled. These settings are mutually exclusive, please choose only one.`
```